src/main/java/bouncybob/util/TaskFileHandler.java

```diff
@@ -0,0 +1,91 @@
+package bouncybob.util;
+
+import bouncybob.task.Deadlines;
+import bouncybob.task.Events;
+import bouncybob.task.Task;
+import bouncybob.task.ToDos;
+
+import java.io.BufferedWriter;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * Utility class for handling file operations related to tasks.
+ */
+public class TaskFileHandler {
+    private static final String FILE_PATH = "./data/bouncy.txt";
```

Comment:
Great work avoiding a common violation of not naming static final variables to the style for constants in the Java Coding Standard!